.editorconfig

```diff
@@ -0,0 +1,43 @@
+root = true
+
+# Global defaults
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Markdown
+[*.md]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = false
+
+# HTML
+[*.html]
+indent_style = space
+indent_size = 2
+
+# JavaScript
+[*.js]
+indent_style = space
+indent_size = 2
+
+# YAML
+[*.{yml,yaml}]
+indent_style = space
+indent_size = 2
+
+# Makefiles (requires tab-based indentation)
+[Makefile]
+indent_style = tab
+
+# CSS
+[*.css]
+indent_style = space
+indent_size = 2
+
+# Go templates
+[*.template]
+indent_style = space
+indent_size = 2
```

Comment:
Lets avoid any style enforcement for go templates, since the exact number of spaces etc. are crucial when laying out CLI text (templates are used in text output)

Reply:
Works for me, i'll pull that out.